src/index.js

```diff
@@ -118,7 +118,8 @@ class Client {
 
       return this.request.postAsync({
         auth: _.pickBy(this.auth, _.identity),
-        body,
+        body: JSON.stringify(body),
```

Comment:
We should make it clear that if users of the client want to keep precision, this should send those values as strings.